test/cli/cli_search_test.py

```diff
@@ -53,8 +53,8 @@ def _get_test_text_resource_content(resource):
     def _get_test_json_resource_content(self, resource):
         return json.loads(self._get_test_text_resource_content(resource))
 
-    def _mock_search_hashes_with_results(self, hashes, hash_type):
-        del hashes, hash_type
+    def _mock_search_hashes_with_results(self, hashes):
```

Comment:
we want to force the user to specify the type of hash. And if they do not specify the type of hash then we will assume sha256. So, I need you to re-add the "hash_type" arg to all of these functions. Then update the docs to tell the user to specify the hash type on the command line. And if no hash type is provided, then we will assume sha256.